util/ssh_client.go

```diff
@@ -29,6 +53,106 @@ func (c *sshClientImpl) ExecCommand(username string, password string, ip string,
 	return string(output), err
 }
 
+func (c *sshClientImpl) UploadFile(username string, password string, ip string, srcFile string, destFile string) error {
+	config := &ssh.ClientConfig{
+		User: username,
+		Auth: []ssh.AuthMethod{
+			ssh.Password(password),
+		},
+	}
+	if !IsIP(ip) {
+		return errors.New("invalid IP address")
+	}
+
+	if IsDir(srcFile) || IsDir(destFile) {
+		return errors.New("Is a directory")
+	}
+	client, err := ssh.Dial("tcp", ip+":22", config)
+	if err != nil {
+		log.Fatal(err)
+		return err
+	}
+
+	conn := (*myssh.Client)(unsafe.Pointer(client))
```

Comment:
Hmm, don't like this `unsafe.Pointer` Business. Is there a way for us to get rid of this?